checkov/kubernetes/checks/resource/k8s/ShareHostIPC.py

```diff
@@ -55,7 +55,8 @@ def scan_spec_conf(self, conf: dict[str, Any]) -> CheckResult:
                 else:
                     return CheckResult.PASSED
             return CheckResult.PASSED
-        return CheckResult.FAILED
```

Comment:
can remove all these lines, not needed anymore

Reply:
@gruebel I don't follow - he removed them, and now it isn't marked as `PASSED` - there is no result. IMO - this should be `PASSED`.. Why just remove the lines?

Reply:
the new default is `PASSED`  so you don't need the lines above it, because only the `if spec["hostIPC"]:` is relevant to change the result to `FAILED`

Reply:
I fixed it to what I meant 🙂